FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -687,97 +687,60 @@ namespace edm {
       if (not ci.skipCurrentProcess() and
           (ci.process().empty() or ci.process() == processConfiguration->processName())) {
         auto productModuleLabel = std::string(ci.label());
-        if (productModuleLabel.empty()) {
-          //this is a consumesMany request
-          for (auto const& branch : conditionalModuleBranches) {
-            //check that the conditional module has not been used
-            if (conditionalModules.find(branch.first) == conditionalModules.end()) {
+        //just a regular consumes
```

Comment:
Maybe this comment is not relevant anymore?

Reply:
Agreed. I removed it. It's just leaves one wondering what is not regular. Thanks.